src/test/librgw_file_nfsns.cc

```diff
@@ -221,7 +221,9 @@ TEST(LibRGW, SETUP_HIER1)
     if (do_create) {
       /* create objects directly */
       std::vector<std::string> obj_names =
-	{"foo/bar/baz/quux",
```

Comment:
@mattbenjamin hmmm... then there is some other problems that lead to ENOENT, will check it.